src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -0,0 +1,340 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import analysis.normed_space.basic
+import topology.sequences
+
+/-!
+# Normed groups homomorphisms
+
+This file gathers definitions and elementary constructions about bounded group homormorphisms
+between normed groups (abbreviated to "normed group homs").
+
+The main lemmas relate the boundedness condition to continuity and Lispschitzness.
+
+The main construction is to endow the type of normed group homs between two given normed group
+with a group structure and a norm (we haven't proved yet that these two make a normed group
+structure).
+
+Some easy other constructions are related to subgroups of normed groups.
+-/
+
+open_locale nnreal big_operators
+
+set_option old_structure_cmd true
+
+/-- A morphism of normed abelian groups is a bounded group homomorphism. -/
+structure normed_group_hom (V W : Type*) [normed_group V] [normed_group W]
+  extends add_monoid_hom V W :=
+(bound' : ∃ C, ∀ v, ∥to_fun v∥ ≤ C * ∥v∥)
+
+attribute [nolint doc_blame] normed_group_hom.mk
```

Comment:
Does it really complain about this one? I've not seen that before

Reply:
You are right. Thanks